yoti-sdk/src/main/java/com/yoti/mobile/android/sdk/YotiSDKButton.java

```diff
@@ -6,24 +6,24 @@
 import android.os.Handler;
 import android.os.ResultReceiver;
 import android.util.AttributeSet;
-import android.view.View;
 
 import androidx.annotation.Nullable;
 import androidx.appcompat.widget.AppCompatButton;
 
-import com.yoti.mobile.android.commons.ui.widget.YotiButton;
 import com.yoti.mobile.android.sdk.exceptions.YotiSDKException;
+import com.yoti.mobile.android.sdk.exceptions.YotiSDKAppNotInstalledException;
 import com.yoti.mobile.android.sdk.exceptions.YotiSDKNoYotiAppException;
 
 /**
  * Custom {@link AppCompatButton} which will start its associated {@link
  * com.yoti.mobile.android.sdk.model.Scenario} when clicked.
  */
-public class YotiSDKButton extends YotiButton implements View.OnClickListener {
+public final class YotiSDKButton extends YotiButtonContainer {
 
     private String mUseCaseId;
     private OnYotiButtonClickListener mOnYotiButtonClickListener;
     private OnYotiAppNotInstalledListener mOnYotiAppNotInstalledListener;
+    private OnAppNotInstalledListener mOnAppNotInstalledListener;
```

Comment:
What's the difference between `OnYotiAppNotInstalledListener` and `OnYotiAppNotInstalledListener`?

Reply:
`OnYotiAppNotInstalledListener` (Deprecated) - listener to notify integrators when the Yoti app is not installed.
`OnAppNotInstalledListener` - notify integrators if app not installed and also we are providing the app url, so that integrator can redirect user to app store for downloading the app.

Reply:
I see! didn't realise that `OnYotiAppNotInstalledListener` is deprecated :)